### ISSUE TITLE
Document "--amo-metdata" json file format

### ISSUE THREAD

**Issue by karlicoss** [mozilla/web-ext#3137]
### TLDR

Is it possible to link directly to the documentation for this metadata json file format/example from `web-ext --help`? 
IMO it's not very clear from the [API docs](https://mozilla.github.io/addons-server/topics/api/addons.html) it currently links to what can you put in the metadata file.

The only example I found across the whole github is this file https://github.com/elvisoliveira/meetings-addon/blob/master/addon-metadata.json

### Slightly longer explanation

I tried using new submission API to upload my extension to AMO. 
The following command worked perfectly for uploading the addon itself:

`web-ext sign --use-submission-api --channel unlisted --source-dir /project/extension/dist/firefox --amo-metadata amo-metadata.json --artifacts-dir /project/extension/dist/artifacts/firefox --api-key <redacted> --api-secret <redacted>`

Then I was wondering whether it's possible to populate the "Notes for Reviewers" field? I often find that I have to copy paste the same instructions over and over when there is a manual review involved, so was hoping to automate this.
The only possibly relevant field I found in [docs](https://mozilla.github.io/addons-server/topics/api/addons.html) is "approval_notes", but this doesn't seem to do anything:

```
$ cat amo-metdata.json
{
  "approval_notes": "just testing if I can pass approval notes here",
  "release_notes": "testing release notes"
}
```

After more search I found this relevant issue https://github.com/mozilla/addons/issues/6158 -- but it's kind of unclear from the it whether it's possible now or not.

Thanks!

### Version information (for bug reports)

```
node --version && npm --version && npm run web-ext -- --version
v20.13.1
10.5.2
7.11.0
```

**Comment by willdurand** [comment]
I will move this bug to the docs repo because https://extensionworkshop.com/documentation/develop/web-ext-command-reference/ should have a more comprehensive section about this file and its content.
